docs_app/content/navigation.json

```diff
@@ -48,7 +48,18 @@
 		{
 			"url": "api",
 			"title": "Reference",
-			"tooltip": "RxJS Reference"
+      "tooltip": "RxJS Reference",
```

Comment:
Somehow locally this file seems to be wrongly formatted. Could you check that?

Reply:
I could only see some weird formatting between lines 29 and 39, but those were ones I hadn't touched.

I ran prettier on the file which made adjustments to those lines, and I'm pushing that change up with my rebase.